README.md

```diff
@@ -1,2 +1,15 @@
-# freighter-backend
+# Freighter-Backend
 Freighter's indexer integration layer and general backend
+
+## Prerequisites
+
+You will need
+
+- Node (>=18.0): https://nodejs.org/en/download/
+- Yarn (>=v1.22.5): https://classic.yarnpkg.com/en/docs/install
+
+## Development
+`npm i && npm run start`
+
+## Production build
+`npm run build:prod`
```

Comment:
just to be symmetric, we should probably use the `yarn` cmd's here

Reply:
fixed in ca473683a27810d0e56bbf4c9e6123e5ae5e8782